stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -101,6 +102,16 @@ class Value():
     def __init__(self, typecode_or_type: str, *args: Any, lock: bool = ...) -> None: ...
 
 # ----- multiprocessing function stubs -----
+def active_children() -> List[Process]: ...
+def allow_connection_pickling() -> None: ...
 def cpu_count() -> int: ...
 def freeze_support() -> None: ...
+def get_all_start_methods() -> List[str]: ...
+def get_context(method: Optional[str] = ...) -> BaseContext: ...
+def get_logger() -> Logger: ...
+def get_start_method(allow_none: Optional[bool] = ...) -> str: ...
```

Comment:
This one (as well as get_all_start_methods and get_context) doesn't exist in Python 2.

Also, judging from the documentation `allow_none` should just be a bool in Python 3, and the return type should be `Optional[str]`.